### ISSUE TITLE
Make price ranges accessible

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1669]
In many blocks we display price ranges:

![imatge](https://user-images.githubusercontent.com/3616980/73350309-0e8a7b80-428d-11ea-90c7-de6e668fffd3.png)
![imatge](https://user-images.githubusercontent.com/3616980/73350334-1a763d80-428d-11ea-8bf1-7fb94066cc6e.png)
![imatge](https://user-images.githubusercontent.com/3616980/73350364-28c45980-428d-11ea-8aeb-b32d55d12780.png)

Currently, screen readers read both prices without specifying what relation is there between them. Instead, it should be read as "From €18,00 to €45,00", or something similar.

The solution we apply for that, should work for all blocks that display prices, including JS-rendered and PHP-rendered blocks.
